### PR TITLE
modules chapter notes , links

### PR DESCRIPTION
**Pull request by aammd**
here is my belated contribution regarding the modules chapter! thanks again everyone for this inspiring group

### CODE REVIEW

File: histogram.R [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+mod_histogram_ui <- function(id){
```
Comment (russHyde) [inline histogram.R]: Could you move this file to `./examples/19-modules/histogram.R`

File: shiny_module_demo.R [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ui <- fluidPage(
+  selectInput("var", "Variable", names(mtcars)),
```
Comment (russHyde) [inline shiny_module_demo.R]: likewise, to `./examples/19-modules/shiny_module_demo.R`

File: test.R [hunk 3]
```diff
@@ -0,0 +1,39 @@
+#' test UI Function
```
Comment (russHyde) [inline test.R]: likewise to `./examples/19-modules/test.R`

**Review by russHyde — CHANGES_REQUESTED** [review]
Thanks so much for this. Could you make sure your example code go into a subdirectory of `./examples/`.

**Review by jonthegeek — APPROVED** [review]
Looks good, thanks!

**Pull request merged**